scripts/extensions/datetimeField/src/getTemplateDateTimeField.tsx

```diff
@@ -0,0 +1,230 @@
+import {ISuperdesk, ITemplateEditorComponentProps} from 'superdesk-api';
+import * as React from 'react';
+import set from 'date-fns/set';
+import format from 'date-fns/format';
+import addMinutes from 'date-fns/addMinutes';
+import {IDateTimeFieldConfig} from './extension';
+import {DatePickerISO, TimePicker, Button, Switch} from 'superdesk-ui-framework/react';
+import {Radio, CheckGroup} from 'superdesk-ui-framework';
+
+interface ITemplateDateTimeFieldState {
+    radioValue: string | null;
+    previousValue: any;
+}
+
+function isDateValue(value: any) {
+    return !!Date.parse(value);
```

Comment:
It's better to avoid using `!!` because it can mean different things. For example, I thought this was a null/undefined check when it's probably not. `Date.parse(value) !== NaN` is much clearer.

Reply:
I thought of the same but then I found out, I cannot use that because suppose the value is a string, so if I use `!!Date.parse(value)` then I would get `false`, but If I use something like this `Date.parse(value) !== NaN` it will return `true`.

Reply:
My bad, I thought that `NaN === NaN` would evaluate to `true`, but it is `false` :D

It should work to use `isNaN(Date.parse('2020-01-01')) !== true`